src/components/ListItem/ListItem.md

```diff
@@ -0,0 +1,63 @@
+# List Item
+Suitable for presenting a summary of an item, with associated actions. Most often used within a List component.
```

Comment:
It might be worth noting, possibly in the associated variants, the history of the ListItem begin tied to email/OWA.